### DISCUSSION TITLE
[Feature] Visited Country Additions: Andorra, Monaco, San Marino, Lichtenstein and Vatican City

### DISCUSSION THREAD

**Discussion by Walmarpole** [johanohly/AirTrail#81] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Unlike AirTrail's flight tracking functionality, the "Visited countries" does not allow selection of any of the above states even when zoomed in and in cases where there are possible airports: Monaco Heliport MCM-LNMC.

Visited countries seems useful as a personal trip history guide part separate from flight history. Addition of the above states would help with that function even in cases without international airports: San Marino, etc.

**Comment by johanohly** [comment]
This is definitely on my radar. The main reason they are not rendered is that using higher definition geo data comes with a significant performance hit. My proposed solution for now is to add a manual entry button, with a list of all countries to choose from (#60).

**Comment by johanohly** [comment]
This has been fixed in v2.4.0!
